geps/gep-91.md

```diff
@@ -16,7 +16,71 @@ This GEP proposes a way to validate the TLS certificate presented by the downstr
 ## Non-Goals
 - Define other fields that can be used to verify the client certificate such as the Cerificate Hash or Subject Alt Name. 
```

Comment:
Can you point out which of the TLS (or mTLS) use cases in https://docs.google.com/document/d/17sctu2uMJtHmJTGtBi_awGB0YzoCLodtR6rUNmKMCs8/edit#heading=h.cxuq8vo8pcxm that are in your goal?

Reply:
Use case number 7, I believe. Are you just asking or suggesting the GEP should include a link?

Reply:
as frank mentioned this is 7, does the doc have to be linked into this GEP ?

Reply:
@arkodg please add a reference to the use case name that's in your goal ( and the doc ).  I strongly believe that tying GEPs in to existing work helps us realize what we've accomplished and what we're still missing from that original list.

Reply:
done